### ISSUE TITLE
Design collapse on folders plus icon

### ISSUE THREAD

**Issue by riceball303** [BoostIO/boost-app-discussions#75]
![cgyamiwusaeqzoe](https://cloud.githubusercontent.com/assets/16876121/14627238/829e54f6-062d-11e6-9526-5169de5f7240.jpg)
![cgyamkduyae0qy7](https://cloud.githubusercontent.com/assets/16876121/14627243/889f8d5c-062d-11e6-8e4e-7f7499078e52.jpg)
It make me always sad. plus sign out of alignment by 1 pixel . Is that only happen on my macbook air?
Excuse for my poor English.

**Comment by Rokt33r** [comment]
Exactly, It is not a intended feature. But that button will be removed from the next version.
Check the screenshot below.

![screen shot 2016-04-19 at 5 47 04](https://cloud.githubusercontent.com/assets/5865853/14633075/d8991d22-0656-11e6-85c0-5363d998b521.png)
